src/rgw/rgw_rest_s3.h

```diff
@@ -26,7 +26,12 @@
 
 #define RGW_AUTH_GRACE_MINS 15
 
-void rgw_get_errno_s3(struct rgw_http_errors *e, int err_no);
+struct rgw_http_error {
+  int http_ret;
+  const char *s3_code;
+};
+
+void rgw_get_errno_s3(struct rgw_http_error *e, int err_no);
```

Comment:
`rgw_get_errno_s3()` doesn't appear to be used anywhere, `struct rgw_http_error` is only used inside